src/loc/record/TokensRecordFrame.tsx

```diff
@@ -65,3 +155,52 @@ export default function TokensRecordFrame(props: { contributionMode?: Contributi
         </Frame>
     );
 }
+
+interface CheckResultProps {
+    state: CheckResult,
+}
+
+function CheckResultFeedback(props: CheckResultProps) {
+    const { state } = props;
+
+    switch (state) {
+        case "POSITIVE":
+            return (
+                <>
+                    <Row className="CheckResultFeedback result-positive" id={ `feedback-${ state }` }>
+                        <Col>
+                            <p>
+                                Check result: <span className="label-positive">positive</span><br />
+                                The Tokens Record - defined by the ID you submitted - is part of the current Collection
+                                LOC.
+                            </p>
+                        </Col>
+                        <Col>
+                            <Icon icon={ { id: "ok" } } height='45px' />
+                        </Col>
+                    </Row>
+                </>
+            )
+        case "NEGATIVE":
+            return (
+                <Row className="CheckResultFeedback result-negative" id={ `feedback-${ state }` }>
+                    <Col>
+                        <p>
+                            Check result: <span className="label-negative">negative</span><br />
+                            The Tokens Record - defined by the ID you submitted - has no match and is NOT part of
+                            the current<br />
```

Comment:
Why a line break here?

Reply:
It's just easier to read (vs a one-line paragraph). And also it's consistent with https://github.com/logion-network/logion-wallet/blob/117b598661a40af8e202ea9d03508936d44f6b3f/src/loc/CollectionLocItemChecker.tsx#L337-L342